File: Currency/src/main/java/ru/fintech/tinkoff/HomeworkApplication.java
package ru.fintech.tinkoff;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.web.reactive.config.EnableWebFlux;

@SpringBootApplication
@EnableCaching
@EnableWebFlux
public class HomeworkApplication {

	public static void main(String[] args) {
		SpringApplication.run(HomeworkApplication.class, args);
	}

}

File: Locations/src/main/kotlin/ru/fintech/kotlin/HomeworkApplication.kt
package ru.fintech.kotlin

import org.springframework.boot.CommandLineRunner
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import ru.fintech.kotlin.datasource.EntityScanner

@SpringBootApplication(scanBasePackages = ["ru.fintech.kotlin"])
class HomeworkApplication: CommandLineRunner {
    override fun run(vararg args: String?) {
        EntityScanner.initialize("ru.fintech.kotlin")
    }
}

fun main(args: Array<String>) {
    runApplication<HomeworkApplication>(*args)
}
